packages/dds/map/src/mapKernel.ts

```diff
@@ -521,14 +539,14 @@ export class MapKernel {
         // Assuming the pendingKeys is small and the map is large
         // we will get the value for the pendingKeys and clear the map
         const temp = new Map<string, ILocalValue>();
-        this.pendingKeys.forEach((value, key) => {
+        for (const [key] of this.pendingKeys.entries()) {
```

Comment:
Can just do this prob
```suggestion
        for (const key of this.pendingKeys.keys()) {
```

Reply:
🤦‍♂️ Sure can. Good catch.